### ISSUE TITLE
README does not correctly reflect state of bufler-tabs-mode

### ISSUE THREAD

**Issue by noprobelm** [alphapapa/bufler.el#97]
I haven't had the time to fully explore this, but I was expecting a `bufler-tabs-mode` global minor mode to be exposed based on the contents of the `README` as well as the `notes.org` associated with this repository.

It seems you might be in the process of rewriting this functionality as indicated by #84 as well as [commit 8bfbcd5](https://github.com/alphapapa/bufler.el/commit/8bfbcd54127f01f812d6e13fa11f55566034fa19). The README should be updated to reflect these changes, and maybe provide additional instructions pointing the user to the fact that they must add `(require 'bufler-workspace-tabs`) to their configuration if they desire to use this feature. 

On another note, in the few moments I used `bufler-workspace-workspaces-as-tabs-mode`, I found I was unable to close any of my tabs at all. I can provide the backtrace if you want it, but I didn't want to inundate this issue with unrelated details and I assumed you're in the process of rewriting those portions of bufler anyway.

I'd offer to make the changes to the `README` myself, but after further review of the current documentation and `bufler-workspace-tabs.el`, I can't say I have a solid enough understanding of how to correctly incorporate `bufler-workspace-workspaces-as-tabs-mode` into a workflow or how to set the relevant configurations appropriately.

**Comment by alphapapa** [comment]
Yeah, sorry.  My to-do list is quite long right now and this isn't a priority.  Some of this functionality has indeed been experimental.  I removed the autoload so that users wouldn't be able to activate it unintentionally.  If one goes out of his way to load it and activate it, he must be interested in exploring, which is okay, too.
